.github/workflows/deployment.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy Hemi Uniswap
+
+on:
+  workflow_dispatch:
```

Comment:
Is the idea to deploy manually?

Reply:
I actually copied this from pure.finance, I didn't even realize it 😅. According to @gabmontes, it doesn't hurt to leave for now.